File: modules/audio-core/src/main/java/com/shterneregen/securelan/audio/service/impl/DefaultAudioProfileService.java
package com.shterneregen.securelan.audio.service.impl;

import com.shterneregen.securelan.audio.service.AudioCallProfile;
import com.shterneregen.securelan.audio.service.AudioProfileService;

public class DefaultAudioProfileService implements AudioProfileService {
    @Override
    public AudioCallProfile defaultProfile() {
        return new AudioCallProfile(48_000, 2, true, true);
    }
}

File: modules/audio-core/src/main/kotlin/com/shterneregen/securelan/audio/service/impl/DefaultAudioProfileService.kt
package com.shterneregen.securelan.audio.service.impl

import com.shterneregen.securelan.audio.service.AudioCallProfile
import com.shterneregen.securelan.audio.service.AudioProfileService

class DefaultAudioProfileService : AudioProfileService {
    override fun defaultProfile(): AudioCallProfile {
        return AudioCallProfile(48_000, 2, echoCancellation = true, noiseSuppression = true)
    }
}
